content/en/news/releases/1.9.x/announcing-1.9.1/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Announcing Istio 1.9.1
+linktitle: 1.9.1
+subtitle: Patch Release
+description: Istio 1.9.1 patch release.
+publishdate: 2020-12-08
+release: 1.9.1
+aliases:
+    - /news/announcing-1.9.1
+---
+
+This release fixes the security vulnerability described in [our March 1st, 2021 news post](/news/security/istio-security-2021-001)
+as well as bug fixes to improve robustness.
+
+This release note describes what’s different between Istio 1.9.0 and Istio 1.9.1.
+
+{{< warning >}}
+Due to the nature of the fixed security issue, at the time of publication qualification tests are still pending. These
```

Comment:
```suggestion
Due to the nature of the fixed security issue, at the time of publication, qualification tests are still pending. These
```

Reply:
I inserted an alternative suggestion.  I don't care about the wording, but I think we should be more specific about what has and has not finished.  So I think we should somehow say that the only shortcut we are taking is only completing 18 out of 48 hours of stress testing.  All the other tests are finished.